.github/workflows/apple.yml

```diff
@@ -47,161 +47,186 @@ jobs:
           - target_triple: 'aarch64-apple-darwin'
             runner: macos-14
             py: 'cpython-3.8'
-            optimizations: 'debug'
+            options: 'debug'
```

Comment:
Initially I added a separate `freethreaded: true` flag but defining all the build options at once makes passing them around much easier.

Reply:
I wonder if this should be `variant`? Or `flavor`, as they're called in the docs? Don't feel strongly.

Reply:
Per the documentation, this isn't the "flavor" it's the "build configuration". The "flavors" are `install_only` / `full`. These are options to configure the build.